_episodes/03-packaging-installing.md

```diff
@@ -52,7 +52,7 @@ Keep motivation as easing in  and gradually scaling up to stay inclusive -->
 
 ## Pip
 
-Pip is the most common package manager for python. Pip allows you to easily install python packages locally from your computer or from an online repository like the [Python Package Index (PyPI)](https://pypi.org/). Once a package is installed with pip, you can `import` that package and use it in your own code.
+Pip is the most common package manager for Python. Pip allows you to easily install Python packages locally from your computer or from an online repository like the [Python Package Index (PyPI)](https://pypi.org/). Once a package is installed with pip, you can `import` that package and use it in your own code.
```

Comment:
is this in two places right now?

Reply:
My bad, that was a bit of an imperfect merge. I pushed ffb1ff7 that fixes it.